models/eos/prefix_lists/eos_prefix_lists.yml

```diff
@@ -0,0 +1,98 @@
+---
+module: eos_prefix_lists
+short_description: Manages Prefix lists resource module
+description: This module configures and manages the attributes of Prefix lists on Arista
+  EOS platforms.
+version_added: 2.1.1
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+notes:
+- Tested against Arista EOS 4.20.10M
+- This module works with connection C(network_cli). See the L(EOS Platform Options,eos_platform_options).
+options:
+   config:
+      description: A list of dictionary of prefix-list options
+      type: list
+      elements: dict
+      suboptions:
+        afi:
+          description:
+          - The Address Family Indicator (AFI) for the  prefix list.
+          type: str
+          required: true
+          choices:
+          - ipv4
+          - ipv6
+        prefix_lists:
+          description:
+          - A list of prefix-lists.
+          type: list
+          elements: dict
+          suboptions:
+            name:
```

Comment:
`name` is common parameters here, so it should be at the same level as `afi` key

Reply:
@justjais There can be multiple names (prefix _lists) under the same afi. Hence it makes sense to have it under afi. Please correct me if I am wrong.

Reply:
@GomathiselviS yes, u can skip my earlier comment. I've implemented ios module in similar way, [ref](https://github.com/ansible-collections/cisco.ios/blob/main/plugins/modules/ios_prefix_lists.py#L246)